spec/models/story_spec.rb

```diff
@@ -19,7 +19,7 @@
 
     specify do
       expect(subject.as_json['story'].keys.sort).to eq([
```

Comment:
Test whether the correct fields are shown or not depending on the story type. Like rendering a view for a release story and see if the not-needed fields are shown, and then rendering a feature story and see if all fields are shown and so on.